### PR TITLE
[VI-638] allow null for rendering map with no base layer

### PR DESCRIPTION
**Pull request by roelandwyns**

### CODE REVIEW

File: src/staticmaps.js [hunk 1]
```diff
@@ -414,6 +414,18 @@ class StaticMaps {
    */
   getTile(data) {
     return new Promise((resolve, reject) => {
+      // custom for rendering map without base layer
+      if( data.url === null ){
```
Comment (OlivierKamers) [inline src/staticmaps.js]: Can you check the lint error? I think it's fixed by doing this:
```
      if (data.url === null) {
        resolve({
          success: true,
          tile: {
            url: data.url,
            box: data.box,
            body: null,
          },
        });
        return;
      }
```

File: src/staticmaps.js [hunk 2]
```diff
@@ -414,6 +414,18 @@ class StaticMaps {
    */
   getTile(data) {
     return new Promise((resolve, reject) => {
+      // custom for rendering map without base layer
```
Comment (OlivierKamers) [inline src/staticmaps.js:417]: Might be nice to add a small test case for it

Comment (roelandwyns) [reply src/staticmaps.js:417]: On this repo or on api?

Comment (OlivierKamers) [reply src/staticmaps.js:417]: This one, let me know if you want to check it on a call

Comment (roelandwyns) [reply src/staticmaps.js:417]: won't do as tests don't run as discussed

File: src/staticmaps.js (hunk 1 above)
Comment (roelandwyns) [reply src/staticmaps.js]: done :)

**Review by OlivierKamers — APPROVED** [review]

**Pull request merged**